nyaa/templates/layout.html

```diff
@@ -275,11 +231,7 @@
 									<option value="0_0" title="All categories" {% if search is defined and search["category"] == "0_0" %}selected{% else %}selected{% endif %}>
 										All categories
 									</option>
-									{% for cat_id, cat_name, cat_title in used_cats %}
-									<option value="{{ cat_id }}" title="{{ cat_title }}" {% if search is defined and search.category == cat_id %}selected{% endif %}>
-										{{ cat_name }}
-									</option>
-									{% endfor %}
+									{{ render_category_options(selected_cat_id=search.category, has_selected_option=search is defined) }}
```

Comment:
I was testing this locally. in the /login page, this crashes because search isn't passed in -- could this be fixed and could you just run through a few scenarios locally to make sure nothing else crashes? thanks!